### ISSUE TITLE
Add support for `<dialog>` element

### ISSUE THREAD

**Issue by foisonocean** [igoradamenko/awsm.css#100]
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog

**Comment by igoradamenko** [comment]
It isn't supported by Safari. Why do you think we should add styles for this element?

**Comment by foisonocean** [comment]
@igoradamenko Yes it's not supported by Safari, but there's a [polyfill](https://github.com/GoogleChrome/dialog-polyfill) for it.
I will use this element sometimes so it would nice if you can add some styles for it.

**Comment by igoradamenko** [comment]
Alright, I think we may add styles for dialog element.

If you wanna you may send a PR, otherwise I will try to implement it myself. I guess in a week or two.
